### DISCUSSION TITLE
MIME-Type Dismatch

### DISCUSSION THREAD

**Discussion by foxbw** [LycheeOrg/Lychee#2507] [Q&A]
I tried to install Lychee according to the instructions. If I call Lychee/public/index.php, I get an error like the image bellow. Something is not displayed correctly here.

![grafik](https://github.com/user-attachments/assets/9965dd20-50f7-42d1-bbce-be93ae90d304)

When I open the dev tools in the browser I get the error message "The resource of "http://.../Lychee/public/index.php/install" was blocked due to a MIME-type mismatch ("text/html") (X-Content-Type-Options: nosniff).

What can I do?

The system runs on a local Ubuntu server with PHP 8.2.

Thanks
Frank

**Comment by ildyria** [comment]
I think your headers/rewrite rules are not set properly.

**Reply by foxbw** [reply]
What do you mean exactly?

I have set in the .htaccess in the directory of Lychee the following code:

`<IfModule mod_headers.c>
    Header always set X-Content-Type-Options "nosniff"
</IfModule>`

In the Dev-Tools the stylesheet document ist set as an "HTML"-File .

![grafik](https://github.com/user-attachments/assets/a1d9903b-0864-491c-8162-0f8e6b824872)

**Reply by ildyria** [reply]
Is this one not working?
https://github.com/LycheeOrg/Lychee/blob/master/public/.htaccess

**Reply by ildyria** [reply]
Did you set up apache (I assume you are on apache) correctly:
https://lycheeorg.github.io/docs/upgrade.html#using-apache

**Reply by foxbw** [reply]
Thank you for the information.
I checked all settings and the installation was successful.

After the installation I see the start page in the browser but I can't do the login. When I click on the login button in the top left corner nothing happens. There are no fields where I can enter my username and password.

**Reply by ildyria** [reply]
Can you check the console log of your browser ? (F12)

**Reply by foxbw** [reply]
livewire/livewire.min.js?id=c4fc8c5d status is 404

Do you need further information?

**Reply by ildyria** [reply]
can you check if livewire/livewire.js exists ?

**Reply by foxbw** [reply]
In the directory

/var/www/html/Lychee/app/Livewire

there ist no livewire.js.

**Reply by ildyria** [reply]
it is generated. Try to access it from your browser.

**Reply by foxbw** [reply]
When I open livewire/livewire.js in the browser, I get the error message “Not Found”.

**Reply by ildyria** [reply]
Okay... Let's drop this.
Go to your `.env` and set `LIVEWIRE_ENABLED=false`

**Reply by foxbw** [reply]
I did the setting.

Now it works.

Thanks
